test/it/optimizer/src/test/resources/converter/select-table.xml

```diff
@@ -18,4 +18,8 @@
 
 <sql-node-converter-test-cases>
     <test-cases sql-case-id="select_from_with_table" expected-sql="SELECT * FROM (TABLE `t0`) AS `dt`" db-types="MySQL" />
+    <test-cases sql-case-id="select_not_in_with_single_table" expected-sql="SELECT * FROM `t_order_item` WHERE `item_id` IS NOT NULL AND `item_id` NOT IN (?, ?) ORDER BY `item_id`" db-types="MySQL" sql-case-types="PLACEHOLDER" />
```

Comment:
Why only test LITERAL?

Reply:
Thank you for your feedback. I will work on adding tests for PLACEHOLDER as soon as possible and update the PR accordingly.